<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SortWidget</class>
 <widget class="QWidget" name="SortWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>276</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>421</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>按选项排序(字典顺序)</string>
   </property>
   <widget class="QRadioButton" name="rb_course_name_">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>课程名字</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_course_number_">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>课程编码</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_study_time_">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>学时与学分</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_for_major_">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>适用专业</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_course_status_">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>课程性质与属性</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_need_major_">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>先修课程</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_manage_">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>课程管理单位</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_teacher_">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>150</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>课程负责人</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pb_no_">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>230</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_yes_">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
